test/assets/wallet0.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"meta":{"generator":"urbit-key-generation-v0.14.1","ship":1,"passphrase":null},"ticket":"~doznec-marbud","shards":["~doznec-marbud"],"ownership":{"seed":"fabric impulse bone solar six turkey midnight loud pizza catalog diet slice problem hover market type light shift response fit fantasy glory member wisdom","keys":{"public":"0253e3cfd74b20ac633d35defa4d421b8a99cc5dbdd180b149fe8bdff8f3947c70","private":"bc484a9d6056138675fd8f24e5c1bdda649c7acb3fcfbc9552fffcf14a20023f","chain":"6544c8c43d875da4b3bb9871d0d2d1b8253dc6a1e2c1666bca865fe18543828b","address":"0x9988685Fc1E5c4079AaEA6A405e0f28956b1bbBe"}},"transfer":{"seed":"pill ensure explain submit rifle street ranch holiday slot carpet memory market stamp asset prefer black promote insane green lemon tornado across stem talent","keys":{"public":"02ba44096a396f1d8c922fdacf564847a1d8e85a6a44d9ba03e162441321e06e8b","private":"c2605506df25f841dc00b43c135965f7ffc95f7e15c198efe59b47bc8fd935e4","chain":"1603e233be5f8c0588ef783eec46d86d88adf253ebab59200073a5bf187e9c48","address":"0x6cB2540C39e673138881744f00C2DFC2382e75c2"}},"spawn":{"seed":"confirm act become tortoise vocal session nuclear enemy auction dirt elephant produce outside symptom involve attack course receive grit solid stand expand brisk ranch","keys":{"public":"023c124789c98e8feaef80e467ad00cfd7597348248b318859813ce3830a221d86","private":"74cbbc06a4339ce8b72ed7e84597d6eb6b2e3d90e672dded5048b2000ed9e33e","chain":"63740b42257aeb76ef539fd208e57bc0d15f2d34b27a99729fab2025a9771387","address":"0xd0f7D56290675609d37FC75F8413535f7aa5DB04"}},"voting":{"seed":"noise vote enter doctor double charge above car tomato brick dutch miracle narrow seed aware harsh bottom armor bonus property census hybrid target symbol","keys":{"public":"028df25e7fabe0b2c74d9d65dcc8007598a560f863498c98b490da420a58e115c1","private":"479fd82a0695563d29b3942bf180f4614a6736f1b731b9df33894bb66a79ca28","chain":"04f0d00f571a833ebd62576424fd084194c3e1488c162bb979227a03fdc38d16","address":"0x38A9e280124b3bA1eaac446643b60752A0663769"}},"management":{"seed":"veteran flash code abuse course front better grace fabric chef puzzle outside visual sauce spice auction cactus galaxy need view tomorrow spirit chimney course","keys":{"public":"03bc9499271df27b4e90cd13a63ba6f48b88aaaca75ed9a7ed97429ef9d7bb4dea","private":"5bed7cc50cb74cd80a798bdb2bc4c7effb7d6875a60d0d0d6ec81efd90fce724","chain":"1135c11e847896630c8bd3c356097f937b6df92d88e2c239f517fa0653ab205a","address":"0x0177441D1b78FaEEEe95c217beBDF02DEB2ce5ba"}},"network":{}}
```

Comment:
Can we maybe stop caring about version numbers for the equality checks on these? Seeing these files change when no crypto-level changes have been made feels not good.

Reply:
Yeah that'd be great, just requires a suitable implementation in the test suite.